src/Ledger/PPUp/Properties.agda

```diff
@@ -4,7 +4,7 @@ open import Ledger.Transaction
 
 module Ledger.PPUp.Properties (txs : TransactionStructure) where
 
-open TransactionStructure txs
+open TransactionStructure txs hiding (Dec-⊎)
```

Comment:
`Dec-⊎` clashes with an instance of `_≤_` that we want Agda to infer in this module.